src/core/HorizontalTabs/HorizontalTabs.tsx

```diff
@@ -202,7 +203,7 @@ export const HorizontalTabs = (props: HorizontalTabsProps) => {
   };
 
   return (
-    <div className='iui-tabs-wrapper'>
+    <div className={cx('iui-tabs-wrapper', 'iui-horizontal')}>
```

Comment:
no need cx

Reply:
it will be needed for next PR (vertical tabs) so I just added beforehand